core/audits/dobetterweb/uses-http2.js

```diff
@@ -105,10 +107,41 @@ class UsesHTTP2Audit extends Audit {
       node.record.protocol = originalProtocol;
     });
 
-    const savingsOnOverallLoad = simulationBefore.timeInMs - simulationAfter.timeInMs;
-    const savingsOnTTI = LanternInteractive.getLastLongTaskEndTime(simulationBefore.nodeTimings) -
+    const savings = simulationBefore.timeInMs - simulationAfter.timeInMs;
+
+    return {
+      // Round waste to nearest 10ms
+      savings: Math.round(Math.max(savings, 0) / 10) * 10,
+      simulationBefore,
+      simulationAfter,
+    };
+  }
+
+  /**
+   * Computes the estimated effect all results being converted to use http/2, the max of:
+   *
+   * - end time of the last long task in the provided graph
+   * - end time of the last node in the graph
+   * @param {Array<{url: string}>} results
+   * @param {Node} graph
+   * @param {Simulator} simulator
+   * @param {{includeLoad?: boolean, providedWastedBytesByUrl?: Map<string, number>}=} options
```

Comment:
We can just drop the options property and treat `includeLoad` as always true in this audit.